src/status_im/contexts/wallet/swap/select_asset_to_pay/view.cljs

```diff
@@ -21,7 +21,9 @@
 (defn- assets-view
   [search-text on-change-text]
   (let [on-token-press (fn [token]
-                         (rf/dispatch [:wallet.swap/start {:asset-to-pay token}]))]
+                         (rf/dispatch [:wallet.swap/start
+                                       {:asset-to-pay           token
+                                        :navigate-within-stack? true}]))]
```

Comment:
Is it possible to give a more meaningful name to `:navigate-within-stack?`? Probably something more related to the effects we desire or the place where it's used?